contract-ui/tabs/claim-conditions/components/snapshot-upload.tsx

```diff
@@ -341,12 +333,16 @@ export const SnapshotUpload: React.FC<SnapshotUploadProps> = ({
                             snapshot
                           </Link>
                         </Text>
-                        <Text as={ListItem}>
-                          Repeated addresses will be removed and only the first
-                          found will be kept.
-                        </Text>
                       </>
                     )}
+                    <Text as={ListItem}>
+                      Repeated addresses will be removed and only the first
+                      found will be kept.
+                    </Text>
+                    <Text as={ListItem}>
+                      The limit you set is for the maximum amount of NFTs a
+                      wallet can claim, not how many they can receive in total.
```

Comment:
this will show for both old and new drops I believe? It's only true for new drops.

Reply:
Well, for old drops is "how many can claim _per transaction_" but the phrase still holds true.

isn't that right?

Reply:
let's add the "per transaction" for the old one, then it's good to ship